### PR TITLE
Document `npm` configuration list options

### LINKED ISSUE #5: Document that you can run `npm config list -l` to get a list of all configuration options
We should put this in readme `npm config list -l`

### PR DESCRIPTION
**Pull request by niftylettuce**
Fixes #5.

### CODE REVIEW

**Comment by sindresorhus** [comment]
No need to also open an issue if you intend to do a PR. PRs *are* issues. This PR also needs a more descriptive title (like the issue). And don't do unrelated changes.

File: readme.md [hunk 1]
```diff
@@ -5,8 +5,8 @@
 
 ## Install
 
-```
-$ npm install npm-conf
+```bash
+npm install npm-conf
 ```
```
Comment (sindresorhus) [inline readme.md]: Don't do unrelated changes. It's like this for a reason. It's not valid `bash` when using `$`.

File: readme.md [hunk 2]
```diff
@@ -24,6 +24,11 @@ conf.get('registry')
 //=> https://registry.npmjs.org/
 ```
 
+To get a list of all available NPM config options:
```
Comment (sindresorhus) [inline readme.md]: NPM => npm

File: readme.md [hunk 3]
```diff
@@ -24,6 +24,11 @@ conf.get('registry')
...
+
+```bash
+npm config list -l
```
Comment (sindresorhus) [inline readme.md]: `-l` => `--long`

**Comment by niftylettuce** [comment]
Sorry @sindresorhus didn't plan on doing a PR at first but realized it was easy enough so I did it anyways.

**Comment by niftylettuce** [comment]
done, updated per your requests, rebased too, @sindresorhus @kevva

**Comment by niftylettuce** [comment]
Oh, actually @sindresorhus I was right, it was `npm config list -l` NOT `npm config list --list`, that does not work.  `--list` is not supported by `npm` there 😦

**Comment by sindresorhus** [comment]
@niftylettuce I wrote `--long`, not `--list`.

**Comment by niftylettuce** [comment]
🤦‍♂️  my bad. fixed!
